include/aspect/compat.h

```diff
@@ -31,5 +31,357 @@
 // for std_cxx14::make_unique:
 #include <deal.II/base/std_cxx14/memory.h>
 
+#if !DEAL_II_VERSION_GTE(9,2,0)
```

Comment:
In other words, put a `//` in front of this line (and its corresponding ones below) and we remove it in a couple of weeks when deal.II dev has the missing functionality.